### PR TITLE
Fce 544/server sdk setup

### PR DESCRIPTION
**Pull request by czerwiukk**
## Description

Describe your changes in detail

### CODE REVIEW

File: guide/server.mdx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    ```
+    poetry add fishjam-server-sdk
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Setup your client
+
+Let's setup everything you need to start communicating with the Fishjam instance.
```
Comment (PiotrWodecki) [inline guide/server.mdx:51]: **nitpick (non-blocking):** 
```suggestion
Let's setup everything you need to start communicating with a Fishjam instance.  
```

File: guide/server.mdx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    poetry add fishjam-server-sdk
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Setup your client
+
+Let's setup everything you need to start communicating with the Fishjam instance.  
+First of all, enter your app in our **developer panel** and copy your **Fishjam URL** and the **Management Token**.
```
Comment (PiotrWodecki) [inline guide/server.mdx:52]: **suggestion (non-blocking):** 
```suggestion
First of all, view your app in the **Fishjam Cloud** developer panel and copy your **Fishjam URL** and the **Management Token**.  
```

File: guide/server.mdx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+    curl -i "$FISHJAM_URL/room" -H $AUTH_HEADER
+    ```
+
+  </TabItem>
+</Tabs>
+
+### Managing rooms
+
+Create room to get the roomId and be able to start adding participants.
```
Comment (PiotrWodecki) [inline guide/server.mdx:96]: **nitpick (non-blocking):** 
```suggestion
Create a room to get the `roomId` and to be able to start adding participants.
```

File: guide/server.mdx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+    # puff, it's gone!
+    ```
+
+  </TabItem>
+</Tabs>
+
+### Managing participants
+
+Create a participant to obtain the participant token for your user to join the room.
```
Comment (PiotrWodecki) [inline guide/server.mdx:144]: **suggestion (non-blocking):** Something like this feels easier to understand
```suggestion
Create a participant to obtain the participant token which will allow your user to join the room.
```

File: guide/server.mdx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+At any time you can terminate his access by deleting the participant.
```
Comment (PiotrWodecki) [inline guide/server.mdx:145]: **issue (blocking):** inclusive language
```suggestion
At any time you can terminate user's access by deleting the participant.
```

File: guide/server.mdx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Server Integration
+
+## Install the SDK
+
+Install the SDK for the language of your choice. We provide libraries for Node, Python and Elixir(?).
```
Comment (PiotrWodecki) [inline guide/server.mdx:12]: **issue (blocking):** Let's not forget about removing this or the question mark itself.
```suggestion
Install the SDK for the language of your choice. We provide libraries for Node, and Python.  
```

**Comment by PiotrWodecki** [comment]
Content-wise, it's good 👍

File: guide/server.mdx [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+    webhook_url = "https://example.com"
+    fishjam_client.add_room(webhook_url=webhook_url)
+    ```
+
+  </TabItem>
+
+  <TabItem value="curl" label="cURL">
+
+    ```bash
+    PAYLOAD='{"webhookUrl": "https://example.com/" }'
```
Comment (PiotrWodecki) [inline guide/server.mdx:211]: **nitpick:** just to match other examples
```suggestion
    PAYLOAD='{"webhookUrl": "https://example.com" }'
```

File: guide/server.mdx [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+    curl -X POST -H "Content-Type: application/json" -H $AUTH_HEADER --data $PAYLOAD  "$FISHJAM_URL/room"
+    ```
+
+  </TabItem>
+
+</Tabs>
+
+### SDK Notifier
+
+Our SDK's come equipped with a Notifier allowing you to subscribe for messages.
```
Comment (PiotrWodecki) [inline guide/server.mdx:221]: **issue (blocking):** 
```suggestion
Our SDKs come equipped with a Notifier allowing you to subscribe for messages.
```

File: guide/server.mdx [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+    ```ts
+    import { FishjamWSNotifier } from '@fishjam-cloud/js-server-sdk';
+
+    const onClose = console.log;
+    const onError = console.error;
+    const onConnectionFailed = console.error;
+
+    const fishjamNotifier = new FishjamWSNotifier({ fishjamUrl, managementToken }, onError, onClose, onConnectionFailed);
+
+    fishjamNotifier.on('roomCreated', console.log);
```
Comment (PiotrWodecki) [inline guide/server.mdx:236]: **issue (blocking):** This example differs in function from the Python one

**Review by PiotrWodecki — COMMENTED** [review]
I would recommend going over this doc with a grammar and spellchecker as we might've not caught all related issues

**Comment by czerwiukk** [comment]
went all over with a spellchecker as well

**Review by PiotrWodecki — APPROVED** [review]

**Pull request merged**